site/.env

```diff
@@ -1,2 +1,2 @@
-DEFAULT_COLLECTION_ID=0
+NEXT_PUBLIC_DEFAULT_COLLECTION_ID=0
 NODE_URL=http://localhost:8545
```

Comment:
should this also be updated to `NEXT_PUBLIC_NODE_URL` ? Or removed if not used?

Reply:
Changed here: https://github.com/hemilabs/pure.finance/pull/1/commits/abcda61f8357cd38f6d8a77b5ddadf6a64b163b9